### PR TITLE
Create recommendations for Houghton

### PR DESCRIPTION
**Pull request by mblack88**
This file will show some really cools things to do in Houghton.
@tcarmean You can add the bike trails.

### CODE REVIEW

File: mblack88-houghton.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# My Recommendation Houghton Michigan  
+## Home Of The Huskies
+
+### Great Places To Eat & Drink
+- Keweenaw Brewing Company (No Food Served, but who needs food)
+- Ambassador (Tostada Pizza)
+- B&B Bar (Pickled Eggs)  
+
```
Comment (edavids2) [inline mblack88-houghton.md:8]: Awesome recomendation

Comment (weiqsun) [reply mblack88-houghton.md:8]: good place to spend summer

Comment (mblack88) [reply mblack88-houghton.md:8]: I even ordered them in priority order as well. :smile:

**Comment by mblack88** [comment]
I am :pray:ing for mine..

**Pull request merged**